### PR TITLE
trace-level supported

### PR DESCRIPTION
**Pull request by pzabolotniy**
hi, @hhkbp2 ! i'd like to suggest to add one more Log-level: Trace (+Tracef). It's less than Debug, but more than NotSet.
We use it in our production applications.

### CODE REVIEW

File: handler_syslog.go [hunk 1]
```diff
@@ -76,6 +76,8 @@ func (self *SyslogHandler) Emit(record *LogRecord) error {
 		err = self.writer.Info(message)
 	case LevelDebug:
 		err = self.writer.Debug(message)
+	case LevelTrace:
+		err = self.writer.Debug(message)
```
Comment (hhkbp2) [inline handler_syslog.go]: Will it be better if we use the default branch for `LevelTrace`?

**Comment by hhkbp2** [comment]
@pzabolotniy Could you add the interfaces `Trace()` and `Tracef()` for root logger too?

File: handler_syslog.go (hunk 1 above)
Comment (pzabolotniy) [reply handler_syslog.go]: Moved to the default branch in commit a9171f8

**Comment by pzabolotniy** [comment]
> @pzabolotniy Could you add the interfaces `Trace()` and `Tracef()` for root logger too?

Appended to the Logger interface, is it proper place?

**Comment by hhkbp2** [comment]
> > @pzabolotniy Could you add the interfaces `Trace()` and `Tracef()` for root logger too?
> 
> Appended to the Logger interface, is it proper place?

The root logger should have `Tracef()` function, like `Debugf()` in

https://github.com/hhkbp2/go-logging/blob/master/init.go#L101

**Comment by pzabolotniy** [comment]
> > > @pzabolotniy Could you add the interfaces `Trace()` and `Tracef()` for root logger too?
> > 
> > 
> > Appended to the Logger interface, is it proper place?
> 
> The root logger should have `Tracef()` function, like `Debugf()` in
> 
> https://github.com/hhkbp2/go-logging/blob/master/init.go#L101

done

**Comment by hhkbp2** [comment]
@pzabolotniy Thanks!

**Pull request merged**